tests/unit/components/WalletAssets.spec.ts

```diff
@@ -1,53 +1,54 @@
-// import Vuex from 'vuex'
-// import { shallowMount, createLocalVue } from '@vue/test-utils'
-
-// import WalletAssets from '@/components/WalletAssets.vue'
-// import { TranslationMock, SoramitsuElementsImport } from '../../utils'
-
-// TODO: fix this issue
-// TypeError: Cannot redefine property: configDescriptor
-//         at Function.defineProperty (<anonymous>)
-//
-//       1 | import axiosInstance from 'axios'
-//     > 2 | import { api } from '@sora-substrate/util'
-
-// const localVue = createLocalVue()
-// localVue.use(Vuex)
-// const store = new Vuex.Store({
-//   modules: {
-//     Account: {
-//       getters: {
-//         accountAssets: () => ([
-//           {
-//             symbol: 'XOR',
-//             address: '1f9840a85d5af5bf1d1762f925bdaddc4201f984',
-//             balance: 12787.09,
-//             decimals: 18
-//           }
-//         ]),
-//         isExternal: () => false
-//       },
-//       actions: {
-//         getAccountAssets: jest.fn()
-//       }
-//     },
-//     Router: {
-//       actions: {
-//         navigate: jest.fn()
-//       }
-//     }
-//   } as any
-// })
-
-describe('WalletAssets.vue', () => {
-  beforeEach(() => {
-    // SoramitsuElementsImport(localVue)
-    // TranslationMock(WalletAssets)
+import Vuex from 'vuex';
+import { shallowMount } from '@vue/test-utils';
+
+import WalletAssets from '@/components/WalletAssets.vue';
+import { AccountAsset } from '@sora-substrate/util';
+import { useDescribe, localVue } from '../../utils';
+import { MOCK_ACCOUNT_ASSETS, MOCK_FIAT_PRICE_AND_APY_OBJECT, MOCK_WALLET_PERMISSIONS } from '../../utils/mock';
+import { WalletPermissions } from '../../../src/consts';
+import type { FiatPriceAndApyObject } from '../../../src/services/types';
+
+const createStore = (permissions: WalletPermissions = MOCK_WALLET_PERMISSIONS, withoutFiatAndApy = false) =>
+  new Vuex.Store({
+    getters: {
+      accountAssets: () => MOCK_ACCOUNT_ASSETS as Array<AccountAsset>,
+      fiatPriceAndApyObject: () => MOCK_FIAT_PRICE_AND_APY_OBJECT as FiatPriceAndApyObject,
+      withoutFiatAndApy: () => withoutFiatAndApy,
+      permissions: () => permissions,
+    },
+  });
+
+useDescribe('WalletAssets.vue', WalletAssets, () => {
+  it('should be rendered correctly', () => {
+    const wrapper = shallowMount(WalletAssets, { localVue, store: createStore() });
+
+    expect(wrapper.element).toMatchSnapshot();
+  });
+
+  it('should not render send button when sendAssets property is false', () => {
```

Comment:
Could you please check here and in test cases below, that buttons exists/not exists with code?
```
const btn = wrapper.find(button)
expect(btn .exists()).toBeTrue();
```